release-notes/10.0/preview/preview3/csharp.md

```diff
@@ -0,0 +1,125 @@
+# C# 14 updates in .NET 10 Preview 3 - Release Notes
```

Comment:
I would add a note at the start or end of each section that we have an incredible place where you can discuss and design the features with us and the rest of hte community, and then link to he corresponding discussions on csharplang.

Reply:
That was the intent, although I did not have the links yet. Using discussion per our email.